src/SitemapServiceProvider.php

```diff
@@ -7,27 +7,29 @@
 
 class SitemapServiceProvider extends ServiceProvider
```

Comment:
We could use the `PackageServiceProvider` from spatie/laravel-package-tools to clean this up

Reply:
https://github.com/spatie/laravel-sitemap/pull/368/commits/6aafa86724d4a3d7bcb719617040b03968154af2